src/plone/z3cform/layout.rst

```diff
@@ -105,22 +104,25 @@ now make the actual call to ``wrap_form`` and register the view class
 it returns.  Normally, you'd register this view class using ZCML, like
 with any other view.
 
+::
+
     >>> from plone.z3cform.layout import wrap_form
     >>> view_class = wrap_form(MyForm, index=layout, label=u'My label')
     >>> provideAdapter(adapts=(Interface, IBrowserRequest),
     ...                provides=Interface,
     ...                factory=view_class,
     ...                name=u'test-form2')
 
-Let's render this view:
+Let's render this view::
 
     >>> view = getMultiAdapter(
     ...     (context, request), name=u'test-form2')
+    >>> view.update(debug=1)
```

Comment:
This seems to be a leftover? :thinking: the Plone Jenkins CI pull request 5.2 job failed only at this point: https://jenkins.plone.org/job/pull-request-5.2/1536/